### ISSUE TITLE
cant run multiple bots

### ISSUE THREAD

**Issue by johnarbuz** [kolton/d2bot-with-kolbot#1887]
Hi guys!

i got 3 cdkey sets related to 3 bot profiles, but when i run them, only first bot log into battlenet.  All remaining diablo windows stuck on "checking versions".

What should i do?
every profile and every cdkey set work very well if lunched alone. 
I dunno why im unable to run more than 1 bot at once

**Comment by mf022** [comment]
please check https://github.com/blizzhackers/documentation/blob/master/kolbot/FAQ.md#how-to-avoid-checking-versions-issue

**Comment by johnarbuz** [comment]
> please check https://github.com/blizzhackers/documentation/blob/master/kolbot/FAQ.md#how-to-avoid-checking-versions-issue

i have that done before

**Comment by Jestah66** [comment]
I had that.  Contacted B-Net support.  It was a two week ban I was told.  I was able to use an alternate interweb source to wait it out.....sure enough two weeks and I could go back.  Changing IP didn't matter, it's like they blocked Comcast.

**Comment by johnarbuz** [comment]
> I had that. Contacted B-Net support. It was a two week ban I was told. I was able to use an alternate interweb source to wait it out.....sure enough two weeks and I could go back. Changing IP didn't matter, it's like they blocked Comcast. Didn't matter what CD-Key I used either.

i tried many cdkey sets. I even bought a fresh one on g2a. 
Bot works well on every cdkey set if it's lunched alone. The problem starts when i open second and third diablo window

**Comment by Jestah66** [comment]
True.....  Checking versions could mean multiple things.  I would try a fresh download/install of the bot and try opening multiple windows with that if you haven't already to rule that out.

**Comment by johnarbuz** [comment]
anyone?

**Comment by mf022** [comment]
Are you running the same IP?
Have you tried to change the IP and see what is happening?

If changing the IP haven't a good result for you, maybe you should try to test another ISP (like mobile connection) in order to see if your issue is ISP related.
